src/reducers/rootReducers.js

```diff
@@ -0,0 +1,34 @@
+const initialState = {
+  films: null,
+  selectedFilm: null
+};
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case "FILM_SELECTED":
+      return { ...state, selectedFilm: action.payload };
+
+    case "FETCH_DATA":
+      return { ...state, films: action.payload.data };
+
+    case "SORT_BY_RATING":
+      return { ...state, films: action.sortBy
```

Comment:
you can use helper function here to avoid copy-paste. This function will take sorting field as a argument and returns sorted array.

Reply:
done